data/upgrades/modification.json

```diff
@@ -535,5 +535,24 @@
     "standard": true,
     "extended": true,
     "epic": true
+  },
+  {
+    "name": "Unstable Sublight Engines",
+    "limited": 0,
+    "xws": "unstablesublightengines",
```

Comment:
TODO here - forgot -battleofyavin suffix, clean up when either we decide all of these cards/abilities should have this tag, or the new names shouldn't have it.